python/ray/dashboard/modules/job/tests/test_http_job_server.py

```diff
@@ -36,6 +36,8 @@
 from ray.runtime_env.runtime_env import RuntimeEnv, RuntimeEnvConfig
 from ray.tests.conftest import _ray_start
 
+import psutil
```

Comment:
this should go before `pytest` (import ordering is builtins, thirdparty, then ray internal)

I think we have a linter that usually enforces this 🤔